package.json

```diff
@@ -54,7 +54,7 @@
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-import": "^2.20.0",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
This was needed to fix the issue reported in this [issue](https://github.com/airbnb/javascript/issues/2331)